src/backend/src/prisma/schema.prisma

```diff
@@ -282,6 +281,28 @@ model Work_Package {
   stage              Work_Package_Stage?
 }
 
+model LinkType {
+  linkTypeId  String   @id @default(uuid())
+  name        String
+  dateCreated DateTime @default(now())
+  creatorId   Int
+  iconName    String
+  creator     User     @relation(name: "linkTypeCreator", fields: [creatorId], references: [userId])
+  links       Link[]   @relation(name: "linkTypes")
+}
+
+model Link {
+  linkId      String   @id @default(uuid())
+  url         String
+  creatorId   Int
+  creator     User     @relation(name: "linkCreator", fields: [creatorId], references: [userId])
+  dateCreated DateTime @default(now())
+  linkTypeId  String
+  linkType    LinkType @relation(name: "linkTypes", fields: [linkTypeId], references: [linkTypeId])
+  projectId   Int
```

Comment:
could be good for work packages too (imagine D&C with like github issue links or something idk)